app/lib/admin/tools/remove_package_from_publisher.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:args/args.dart';
+import 'package:clock/clock.dart';
+
+import 'package:pub_dev/package/backend.dart';
+import 'package:pub_dev/package/models.dart';
+import 'package:pub_dev/publisher/backend.dart';
+import 'package:pub_dev/shared/datastore.dart';
+
+final _argParser = ArgParser()
+  ..addOption('package', help: 'The package to update.')
+  ..addOption('publisher-id', help: 'The `publisherId` to set.')
+  ..addFlag('help', abbr: 'h', defaultsTo: false, help: 'Show help.');
+
+Future<String> executeRemovePackageFromPublisher(List<String> args) async {
```

Comment:
Would be nicer to make admin actions:
https://github.com/dart-lang/pub-dev/blob/master/app/lib/admin/actions/actions.dart

We don't have CLI for admin-actions yet, but it should be easy to build.

Reply:
Ah yes, I forgot you landed these. Much nicer!

Done